etc/singularity.conf

```diff
@@ -17,8 +17,30 @@ allow setuid = yes
 # Should we allow users to request the PID namespace?
 allow pid ns = yes
 
-
-# ENABLE OVERLAY: [BOOL]
+# ALLOW USER IMAGE: [BOOL]
+# DEFAULT: yes
+# Should we allow users to run images they own?  If set to 'no', then the image
+# can only be executed through 'protected image mode' (see below).
+#
+# Note that if 'allow user image' is no and 'protected image mode' is none,
+# then no image file may be used.
+allow user image = yes
+
```

Comment:
ok, so this is sort of like a controller for extending permissions for singularity image? For example, by quickly changing the values in the config file (protected image mode to none and allow user image to no) then no containers can be used. This would be very useful in some kind of situation where singularity needed to be "frozen" but not uninstalled / removed. 

Are these permissions living / changing or do they apply to all images created after being set?